codecoverage.py

```diff
@@ -52,7 +52,13 @@ def download_artifact(task_id, artifact):
         shutil.copyfileobj(r.raw, f)
 
 
-def download_coverage_artifacts(build_task_id):
+def suite_name_from_task_name(name):
+    name = name[len('test-linux64-ccov/opt-'):]
+    parts = [p for p in name.split('-') if p != 'e10s' and not p.isdigit()]
+    return '-'.join(parts)
+
+
+def download_coverage_artifacts(build_task_id, suites):
     try:
         shutil.rmtree("ccov-artifacts")
```

Comment:
I think there's an ignore_errors parameter for rmtree, so you don't have to have a try block around it.